packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1013,17 +1013,26 @@ export class ProjectService {
         user: SessionUser,
         chartUuid: string,
     ): Promise<Record<string, any>[]> {
-        const chart = await this.savedChartModel.get(chartUuid);
-        const { metricQuery } = chart;
-        const exploreId = chart.tableName;
-
-        const rows = await this.runMetricQuery(
-            user,
-            metricQuery,
-            chart.projectUuid,
-            exploreId,
-            undefined,
-            QueryExecutionContext.GSHEETS,
+        const rows = await wrapSentryTransaction(
+            'getResultsForChartWithWarehouseQuery',
+            {
+                user,
```

Comment:
should we only use userUuid , and/or name instead of the full user object ? I'm not sure if sentry can even serialize that

Reply:
`CustomSamplingContext` accepts a `data` property that is an object of key `string`s and value `any`, so I would like to keep this to see what it does 👍 
<img width="1016" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/381ca3af-025d-41dc-be50-7d1cd27ef26c">

Reply:
> should we only use userUuid

yes, took your suggestion, thank you!